tests/autoyast/prepare_profile.pm

```diff
@@ -31,8 +32,10 @@ use autoyast qw(
 sub run {
     my $path = get_required_var('AUTOYAST');
     # Get file from data directory
-    my $profile = get_test_data($path);
-
+    my $profile  = get_test_data($path);
+    my $testdata = get_test_suite_data();
+    # if testdata formula means that salt-formulas will be used
+    set_var("FORMULAS_PATH", data_url($testdata->{formulas})) if $testdata;
```

Comment:
Not to set variables everywhere, I would propose to set it in the profile as a variable. As we don't have any other data there, we won't loose much in structure.
As an alternative, we can change `expand_variables` method to support testdata and expand variables there from some special key, e.g. `autoyast->vars`. WDYT?

Reply:
>As an alternative, we can change expand_variables method to support testdata and expand variables there from some >special key, e.g. autoyast->vars

I like this idea, i just think that it could be a separate ticket because it goes some extra miles on the task.

i will have to try the first approach for now, that it didnt work for me and see if i can fix it

Reply:
Having a second thought about the integration of the testdata into the expand_variables, i think that the function should be kept simple doing one thing well as it does. if i would change something it would be maybe the expansion of the @vars using the testdata.